Makefile

```diff
@@ -1,6 +1,12 @@
 SHELL := /bin/bash
 ENABLE_FEATURES ?= default
 
+ifeq ($(TCMALLOC),1)
+ENABLE_FEATURES = tcmalloc
+else
+ENABLE_FEATURES = jemalloc
```

Comment:
L2 seems useless.

Reply:
It seems that this is intended to be appended to `ENABLE_FEATURES` instead?

Reply:
@Hoverbear  `+=` ?

Reply:
Yeah they should be `+=`. But also, I missed something here in my last round of review.

The way these features are set up, the makefile needs to turn off default features, then turn on the features that it wants. But line 2 is still initializing `ENABLE_FEATURES` to `default` if it isn't already set.

So in addition to these assignments being compound `+=` assignments, L2 needs to say `ENABLE_FEATURES ?=`. That is, if the environment contains `ENABLE_FEATURES` use that value, if not, initialize it to empty.

That raises another issue though, that if the user does try to set `ENABLE_FEATURES`, then the makefile will add its own features in anyway; so the user better not specify any features that conflict with the makefile. This is a pre-existing issue though so we don't need to solve it.

Please change these two assignments to `+=` and line 2 to `ENABLE_FEATURES ?=`.